src/commands/mod.rs

```diff
@@ -94,10 +99,12 @@ impl Command {
         match self {
             Self::Key2ds(key2ds) => key2ds.execute(env),
             Self::Keygen(keygen) => keygen.execute(env),
+            Self::Keyset(keyset) => keyset.execute(env),
             Self::Nsec3Hash(nsec3hash) => nsec3hash.execute(env),
             Self::Notify(notify) => notify.execute(env),
             Self::SignZone(signzone) => signzone.execute(env),
             Self::Update(update) => update.execute(env),
+            // Self::Help(help) => help.execute(env),
```

Comment:
Nit but I think this stems from a conflict.

Reply:
As far as I know, help was disabled by Jannik until the help command can render man pages.

Reply:
Ah, I see, in commit 6915b816a65dbafa1ed27f0cd84781362dbb7f0d:
-            // Self::Help(help) => help.execute(env),
+            Self::LdnsUpdate(ldnsupdate) => ldnsupdate.execute(env),

That was just too confusing.